pkgs/stdenv/linux/default.nix

```diff
@@ -93,13 +93,31 @@
   files = archLookupTable.${localSystem.system} or (if getCompatibleTools != null then getCompatibleTools
     else (abort "unsupported platform for the pure Linux stdenv"));
   in files
+, enableGccExternalBootstrapForStdenv ? true
```

Comment:
This needs a comment what `enableGccExternalBootstrapForStdenv` is supposed to do.